pkg/cmd/cli/cli.go

```diff
@@ -34,9 +34,9 @@ To create a new application, you can use the example app source. Login to your s
 run new-app:
 
   $ %[1]s login
-  $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
+  $ %[1]s new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-hello-world.git
```

Comment:
and the existing text wasn't wrong, i just figured it would be nice for the example to point people to the current image.